src/components/FeatureViews/AdminPortal/CreateTopicModal.js

```diff
@@ -90,57 +95,44 @@ const GET_COURSE_TAGS = gql`
     }
 `;
 
-const CreateTagModal = ({ closeModal }) => {
-    const [courseTagData, setCourseTagData] = useState({
-        tagName: '',
-        tagDescription: '',
+const CreateTopicModal = ({ closeModal }) => {
+    const [courseTopicData, setCourseTopicData] = useState({
```

Comment:
refactor "courseTopicData" to "courseTopicValue". Data = database, Value = state values